SS_popdyn.tpl

```diff
@@ -107,13 +107,13 @@ FUNCTION void get_initial_conditions()
   }
   #ifdef DO_ONCE
   if (do_once == 1)
-    cout << " MG setup OK " << endl;
+    echoinput << " MGsetup OK " << endl;
```

Comment:
There is some inconsistency throughout the files in this pull request regarding the spacing, e.g.,
```
echoinput << " text" << endl;
```
versus
```
echoinput << "text" << endl;
```
There could be a reason for this; but, here MGsetup and growth1 have a pre-space but natmort doesn't.